algorand/go-algorand
# DevOps: alphanet master remerge

- Bump version in buildnumber.dat
- Empty box hotfix for relstable (#16)
- network: increase max incoming connections limit to 2400 (#5018)
- CI: Fix PartitionTest Github Action (#5006)
- Tests: Fix t.Parallel() errors in shared package (#4989)
- network: new default gossip fanout setting for relays (#5005)
- algocfg: clarify config file loading error (#5025)
- agreement: fix receivedAt proposal performance timing (#5041)
- Update the Version, BuildNumber, genesistimestamp.data
- Algod: Replace `logic.DebuggerHook` with `logic.EvalTracer` interface (#4438)
- Txhandler: Improve dedupe hit rate (#5022)
- lint: fix warnings (#5047)
- Fix opBytesLt for len(rhs) < len(lhs) (#5051)
- tests: disable TestInitialSync on CI (#5053)
- Update the Version, BuildNumber, genesistimestamp.data
- CICD: Adding github release notes template (#5044)
- build: add -Wno-deprecated to sortition (#5050)
- Fix the inline comment for proposalAccepted (#4889)
- development: tool to convert validated blocks. (#5048)
- e2e tests: log pending transactions in case of failure (#5062)
- build: add start anchor to paralleltest exclusion regex (#5046)
- goal: allow relative dataDir via -d cmd option (#5067)
- build(deps): bump github.com/aws/aws-sdk-go from 1.16.5 to 1.33.0 (#4955)
- docs: fix typo in avm/teal spec (#5074)
- goal: add --start option for network create command (#4902)
- ledger: abstract store and remove all db.Atomic direct usages (#5021)
- ledger: remove sql.Tx from the transaction callback  (#5031)
- network: Add GetUnderlyingConnTcpInfo method to wsPeer (#5077)
- tests: add more logging to expect test (#5078)
- goal: resolve data dir code duplication (#5073)
- node: Add follower node for sync mode (#5009)
- goal: allow ConfigJSONOverride in local network templates (#5017)
- tests: reenable catchpoint tests (#4419)
- algod: split SetFdSoftLimit calls for relay and non-relay nodes (#5070)
- container: support kmd (#4984)
- network: Ignore invalid tags (#4517)
- goal: goal network create default templates (#4891)
- ledger: turn deferredCommitContext.newBase into a function (#5093)
- link to security doc, minor formatting (#5095)
- Bump buildnumber.dat
- algod: split SetFdSoftLimit calls for relay and non-relay nodes (#5070)
- kv delta fix (#5084)
- Rename enums by default (#5089)
- fix for goal node status crash - no longer getting block (#5100)
- fix for goal node status crash - no longer getting block (#5100)
- fix: disable update-repo-description (#5096)
- tests: fix TestDelta by sorting results (#5103)
- e2e-test: Fix follower e2e test (#5114)
- Bump Version, Remove buildnumber.dat and genesistimestamp.dat files.
- node: Fix time since last block. (#5113)
- tests: debug flaky expect test (#5119)
- algod: Refactor AccountData conversion (#5098)
- AVM: Teal macros (#4737)
- txHandler: move streamverifer out of txn go (#5039)
- lint: check for references to loop variables (#5105)
- Update state delta type, handler (#5123)
- cmd: fix typo in relayCmd.go (#5135)
- add an Enhancement issue-template (#5091)
- tools: remove vbconvert (#5133)
- CircleCI: Fix cached Results issue in self-hosted runner (#5137)
- Algod: Modify `EvalTracer` design and improve testing for failures (#5071)
- Tests: No Cache Testing in ledger (#5058)
- network: connection deduplication (#4695)
- refactor: remove sql.Tx from Batch (#5080)
- network: add OutgoingMessage disconnectReason field (#5147)
- tests: use assert.Eventually to wait for peering to settle (#5150)
- CI: upgrade test_nightly infra to large (#5054)
- tests: fix TestStreamVerifierPoolShutdown (#5152)
- perf: Separate the txn specific operations from StreamVerifier (#5132)
- tests: fix debug output in expect test harness (#5158)
- Algod: Additional simulation result information (#4439)
- tests: guard addPeer from adding peers who are closing (#5151)
- devmode: Allow DevMode + Follower configurations. (#5157)
- algocfg: profile subcommand (#5069)
- ledger: cleanup the store package to prepare for the kv impl  (#5139)
- fix(follower): update test that defines a follower relay. (#5162)
- fix(follower): add experimental tag to deltas endpoint. (#5169)



## Summary



## Test Plan
